Sources/SKSupport/AsyncQueue.swift

```diff
@@ -169,6 +169,15 @@ extension AsyncQueue where TaskMetadata == Serial {
     return self.async(priority: priority, metadata: Serial(), operation: operation)
   }
 
+  /// Executes the given `operation` and awaits its result.
+  @discardableResult
+  public func sync<Success: Sendable>(
```

Comment:
```suggestion
  public func asyncResult<Success: Sendable>(
```
Maybe? Seems weird to call it `sync` since it is really still async.

Reply:
I removed it and added `await <xxx>.value` instead. Turns out that one of the three calls benefited from using `await <xxx>.valuePropagatingCancellation`